### ISSUE TITLE
Fix strategies

### ISSUE THREAD

**Issue by jamesonquinn** [The-Center-for-Election-Science/vse-sim#7]
- Make strategies independent of systems, so you can mix-and-match
- Include a Borda-type strategy
- Include a Condorcet-type strategy

**Comment by wolftune** [comment]
Indeed. It should include the strategies people most often are concerned about. That includes bullet-voting not only for favorite of top-two but bullet-voting for honest favorite (being dishonest about others), polarizing in score (i.e. those who treat score like approval) when done unevenly, and maybe others. Also, non-normalized score voting (e.g. I hate them all, give everyone a low score, but some higher than others still) — as in if score is like stars for a movie, a set of candidates could have no top-scoring candidate…
